node_modules/oae-core/tenantlandingpage/css/tenantlandingpage.css

```diff
@@ -0,0 +1,159 @@
+/*!
```

Comment:
There should not be an empty properties file for this widget. If there are no keys, the file should be removed and the widget manifest should be updated